libs/src/sdk/api/chats/ChatsApi.ts

```diff
@@ -476,15 +483,25 @@ export class ChatsApi
 
   private async decryptLastChatMessage(c: UserChat): Promise<UserChat> {
     const sharedSecret = await this.getChatSecret(c.chat_id)
+    let last_message = ''
+    if (c.last_message && c.last_message.length > 0) {
+      try {
+        last_message = await this.decryptString(
+          sharedSecret,
+          base64.decode(c.last_message)
+        )
+      } catch (e) {
+        console.error(
+          "[audius-sdk]: Error: Couldn't decrypt last chat message",
```

Comment:
Couldn't think of a great way to capture this error in a way that the client could handle it more gracefully and still have the chats/messages that succeeded load

Reply:
Yeah this seems like a good option... hopefully it's a rare thing.